lib/components/Input/input-phone-number/__tests__/__snapshots__/input-phone-number.test.tsx.snap

```diff
@@ -68,9 +68,8 @@ exports[`InputPhoneNumber Component should match the snapshot 1`] = `
             <input
               class="input peer input__align--left input__size--lg"
               id="quill-input-1"
-              inputmode="numeric"
```

Comment:
why did we remove inputmode?

Reply:
This is from the snapshot for the test case, maybe because we are not using the `<PatternFormat />` from `react-number-format` that's why it removes the `inputmode="numeric"` here. I'll add a props here to make it `inputmode="numeric"` again 🙏